dev/CommandBarFlyout/CommandBarFlyout_themeresources.xaml

```diff
@@ -107,6 +108,7 @@
 
     <Thickness x:Key="CommandBarFlyoutAppBarButtonInnerBorderMargin">2</Thickness>
     <Thickness x:Key="CommandBarFlyoutAppBarEllipsisButtonInnerBorderMargin">2,2,6,2</Thickness>
+    <contract12Present:ThemeShadow x:Key="OverflowShadow"/>
```

Comment:
This theme resource will be available globally in apps consuming our package, so you'll want to give it a more descriptive name.  Perhaps `CommandBarFlyoutOverflowShadow`?

Reply:
agreed.

---
In reply to: [612708703](https://github.com/microsoft/microsoft-ui-xaml/pull/4743#discussion_r612708703) [](ancestors = 612708703)